### DISCUSSION TITLE
2.1.1

### DISCUSSION THREAD

**Discussion by eschutho** [apache/superset#25116] [General]
Hello Community,

The Apache Superset team is pleased to announce that Superset
2.1.1 has just been released.

Apache Superset is a modern, enterprise-ready business intelligence web application

The official source release:

https://www.apache.org/dist/superset/2.1.1

The PyPI package:

https://pypi.org/project/apache-superset/

If you have any usage questions or have problems when upgrading or
find any issues with enhancements included in this release, please
don't hesitate to let us know by sending feedback to this mailing
list.

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/2.1.1'>2.1.1</a>.</em>

**Comment by trixieboyle** [comment]
> Hello Community,
> 
> The Apache Superset team is pleased to announce that Superset
> 2.1.1 has just been released.
> 
> Apache Superset is a modern, enterprise-ready business intelligence web application
> 
> The official source release:
> 
> https://www.apache.org/dist/superset/2.1.1
> 
> The PyPI package:
> 
> https://pypi.org/project/apache-superset/
> 
> If you have any usage questions or have problems when upgrading or
> find any issues with enhancements included in this release, please
> don't hesitate to let us know by sending feedback to this mailing
> list.
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/2.1.1'>2.1.1</a>.</em>

**Comment by trixieboyle** [comment]
> Hello Community,
> 
> The Apache Superset team is pleased to announce that Superset
> 2.1.1 has just been released.
> 
> Apache Superset is a modern, enterprise-ready business intelligence web application
> 
> The official source release:
> 
> https://www.apache.org/dist/superset/2.1.1
> 
> The PyPI package:
> 
> https://pypi.org/project/apache-superset/
> 
> If you have any usage questions or have problems when upgrading or
> find any issues with enhancements included in this release, please
> don't hesitate to let us know by sending feedback to this mailing
> list.
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/2.1.1'>2.1.1</a>.</em>

Tammy Boyle
